src/ucp/proto/proto.c

```diff
@@ -14,3 +14,8 @@
 const ucp_proto_t *ucp_protocols[UCP_PROTO_MAX_COUNT] = {};
 unsigned ucp_protocols_count                          = 0;
 
+const char *ucp_perf_type_names[] = {
+    [UCP_PROTO_PERF_TYPE_SINGLE] = "single",
```

Comment:
reuse this array also in:
ucp_proto_select_init_trace_caps()
ucp_proto_select_param_str()